### PR TITLE
Second branch

### PR DESCRIPTION
**Pull request by muskanbagrecha**
Added product listing page. Please review.

### CODE REVIEW

File: css/shop.css [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+}
+
+.slider__value--left {
+    left: 0;
+    bottom: -20px;
+}
+
+.slider__value--right {
+    right: 0;
+}
```
Comment (rahulyadav139) [inline css/shop.css:102]: You have used BEM naming convention for class which is pretty neat. Class names are descriptive and easy to understand. Files sre readable and clean. Great work. Go ahead and merge the branch.

**Review by rahulyadav139 — APPROVED** [review]

File: css/shop.css [hunk 2]
```diff
@@ -0,0 +1,174 @@
+/* Layout */
+.container {
```
Comment (manojsarna) [inline css/shop.css:2]: do check the branch name you have created

File: css/shop.css [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+footer {
+    grid-area: footer;
+}
+
+/* Filter styling */
+
+.filter__container hr {
+    margin: 0.8rem auto;
+    border: 1px solid #ddd;
+}
```
Comment (manojsarna) [inline css/shop.css:31]: can use color vars

File: css/shop.css [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+.filter__header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 0.5rem;
+}
```
Comment (manojsarna) [inline css/shop.css:38]: can use px also if value is less than 10px

File: shop.html [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+                            <input type="checkbox" name="category" id="bridal-category">
+                            <span>Bridal</span>
+                        </label>
+
+                        <label for="cosmetics-category">
+                            <input type="checkbox" name="category" id="cosmetics-category">
+                            <span>Cosmetics</span>
+                        </label>
+
+                        <label for="nameB">
```
Comment (PuneetP16) [inline shop.html:137]: Use more specific name instead of "nameB"

File: css/shop.css [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+    justify-content: space-between;
+    padding: 0.5rem;
+}
+
+.filter__section fieldset {
+    display: flex;
+    flex-direction: column;
+    border: 0;
+    padding: 1rem 0;
+}
```
Comment (manojsarna) [inline css/shop.css:60]: looks good!!

**Review by j24m — APPROVED** [review]
Hello, love your code, because it is so so readable, class names are really good , And the UI looks so clean as well, consistent margin and padding everywhere, nothing looks out of place. Great work.

File: css/shop.css [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+.filter__section input[type="checkbox"] {
+    margin-right: 0.8rem;
+    cursor: pointer;
+}
+
+/* Filter slider for price */
+
+.slider {
+    appearance: none;
+    background: #d3d3d3;
```
Comment (PuneetP16) [inline css/shop.css:78]: use variables instead of values in color.

File: shop.html [hunk 8]
```diff
@@ -0,0 +1,360 @@
...
+                        <label for="threestar">
+                            <input type="radio" name="price" id="threestar">
+                            <span>3 star & above</span>
+                        </label>
+                        <label for="threestar">
+                            <input type="radio" name="price" id="threestar">
+                            <span>2 star & above</span>
+                        </label>
+                        <label for="threestar">
+                            <input type="radio" name="price" id="threestar">
```
Comment (PuneetP16) [inline shop.html:178]: change for and id values to its specific radio input element.

File: css/shop.css [hunk 9]
```diff
@@ -0,0 +1,174 @@
...
+.filter__section .filter__section-title {
+    font-size: 1.15rem;
+    font-weight: 500;
+    color: var(--color-dark);
+}
+
+.filter__section input[type="radio"],
+.filter__section input[type="checkbox"] {
+    margin-right: 0.8rem;
+    cursor: pointer;
```
Comment (manojsarna) [inline css/shop.css:71]: check for px vlaues

**Review by PuneetP16 — APPROVED** [review]
very good work on ui, after necessary changes good to merge.

File: css/shop.css [hunk 10]
```diff
@@ -0,0 +1,174 @@
...
+}
+
+/* Product Listings */
+.products__list-container {
+    grid-area: products;
+    place-content: center;
+    padding: 3rem;
+    margin-left: auto;
+    display: flex;
+    flex-wrap: wrap;
```
Comment (manojsarna) [inline css/shop.css:124]: BEM Model Usage looks good!!

File: shop.html [hunk 11]
```diff
@@ -0,0 +1,360 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+
+    <!-- CSS -->
+
+    <!-- Component Library -->
+    <link rel="stylesheet" href="https://card-input.netlify.app/lighttheme.css">
+
+    <!-- Custom CSS -->
+    <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" href="css/shop.css">
```
Comment (manojsarna) [inline shop.html:20]: uneven spacing

File: shop.html [hunk 12]
```diff
@@ -0,0 +1,360 @@
...
+        <nav class="navigation navigation-full">
+
+            <!-- logo -->
+            <a href="index.html">
+                <img src="./assets/logo.png" alt="logo" class="logo">
+            </a>
+
+            <!-- menu -->
+            <ul class="nav-list">
+                <li><a href="index.html">Home</a></li>
```
Comment (manojsarna) [inline shop.html:36]: use Beautify

**Review by manojsarna — CHANGES_REQUESTED** [review]
Overall Looks Good!!

Do Check The Comments!!

**Pull request merged**